index.js

```diff
@@ -170,15 +170,20 @@ module.exports = fp(async function (app, opts) {
       }
     }
 
-    // Execute
-    // This may throw. Where is the error going?
```

Comment:
Note this comment. I probably didn't know what I was doing!

Reply:
yes the way to handle it was not obvious :)